src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java

```diff
@@ -113,7 +114,8 @@ public void getTasksBySearchForCompletableCriteria() {
     private void setInitMockForSearchByCompletableTask() {
         AccessControlResponse accessControlResponse = mock((AccessControlResponse.class));
         when(accessControlService.getRoles(anyString())).thenReturn(accessControlResponse);
-        when(camundaService.searchForCompletableTasks(any(), any(), any())).thenReturn(createTasks());
+        when(camundaService.searchForCompletableTasks(any(), any(), any()))
+            .thenReturn(new GetTasksCompletableResponse<>(false, createTasks()));
```

Comment:
Can we add a local consumer test for this endpoint? The response has changes to include a (boolean, List<Task>). Existing tests expected Task or Task so let's make sure;
1. The local consumer verifies the correct expected structure following this change.
2. ExUI have a point of reference if the change is breaking and they have to update their consumer pact.
There are example local consumers  especially for GetTask which can be reused. Give me a shout if you need more info.